### PR TITLE
Update tutorials for Pupil v2.0

### PR DESCRIPTION
**Pull request by pfaion**
Upgrade all tutorials to work with Pupil v2.0 output.

Summary of Changes:
- Replaced sample_recording with apriltag-demo-recording with a v2.0 export (called **sample_recording_v2**)
- Replaced code to download the surface reference image by placing the image (for the new recording) in the `data/` folder.
- Removed all examples for multi-subject aggregation since the new sample_recording is only one file (we did not have the reference recordings anymore that were used to generated these tutorials in the past anyways).
- Filtered pupil data always for `method == "3d c++"`
- Adjusted all text for changes to v2.0
- Fixed all broken links
- Fixed broken tutorial05 (sample data file was empty somehow)
- Clean rerun all notebooks, improved some prints

### CODE REVIEW

**Review by willpatera — CHANGES_REQUESTED** [review]
As you can see from the commits above :point_up: I have made a few changes; mostly stylistic changes to text and some typo fixes. 

Notes:
1. The last figure in tutorial 01 is a bit confusing - which colors are fixations? 
2. The last figure of scanpath plot in tutorial 03 could use improvement. It is difficult to read any of the numbers. Perhaps one could show a color map for the polyline so you can see the progression. Or cluster the closer fixations (but this would be a lot of work).

File: 01_load_exported_data_and_visualize_pupillometry.ipynb [hunk 1]
```diff
@@ -11,73 +11,49 @@
     "1. Visualize pupil diameter and pupil location as a function of time.\n",
```
Comment (pfaion) [inline 01_load_exported_data_and_visualize_pupillometry.ipynb:11]: <p>@willpatera I think the output here got removed.</p><p>I would probably clean rerun all notebooks in the end once more, just to make sure that everything works fine and outputs match the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pupil-labs/pupil-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (pfaion) [inline 01_load_exported_data_and_visualize_pupillometry.ipynb:11]: <p>Is there a reason for changing these?</p><p>The old tutorials use <code>0.95</code>.</p><p>Below we also still use <code>0.95</code>  for filtering.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pupil-labs/pupil-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (willpatera) [reply 01_load_exported_data_and_visualize_pupillometry.ipynb:11]: <p>@pfaion please could you re-run the notebooks and make another commit today?</p>

Comment (willpatera) [reply 01_load_exported_data_and_visualize_pupillometry.ipynb:11]: <p>@pfaion - please revert this to <code>0.95</code>  and commit. </p>

Comment (pfaion) [reply 01_load_exported_data_and_visualize_pupillometry.ipynb:11]: Done.

Comment (pfaion) [reply 01_load_exported_data_and_visualize_pupillometry.ipynb:11]: Done.

**Comment by pfaion** [comment]
@willpatera Should be fine now.
I did not touch the scanpath as I think it might be a more realistic visualization as before.
Actually I think the previous visualization might have been based on a different version of the fixation detector without a maximum fixation duration (e.g. our online version in capture, or some older version).

**Review by willpatera — APPROVED** [review]
We can merge this. We might want to add back multi-recording analysis as separate tutorials in the future though.

**Comment by pfaion** [comment]
@willpatera I created a task for creating the new tutorials in CU.

**Pull request merged**